### ISSUE TITLE
Seemingly false positive outdated PBO errors

### ISSUE THREAD

**Issue by gmantampa** [acemod/ACE3#5967]
**Arma 3 Version:** `x.xx` (stable / rc / dev)
**CBA Version:** `3.x.x` (stable / dev + commit hash)
**ACE3 Version:** `3.x.x` (stable / dev + commit hash)

**Mods:**
```
- ace
```

**Description:**
- error code file_ace_arsenal.pbo outdated

**Where did the issue occur?**
-eden editor, am unable to play, spectate, in the editor mode. error code pops up and takes me to a end game screen

i no nothing about code and relatively new to mods in gaming for arma. I have no clue what this error code means. and i have deleted and reinstalled mod, but same issue persist. any detailed suggestions, please dumb it down as best as you can for me

**Comment by Cuel** [comment]
If you fill out the issue form correctly it's more likely you'll get help. Also upload the RPT file

**Comment by SilentSpike** [comment]
Are you using the workshop version?

**Comment by Triiick** [comment]
Also having this issue, and i see multiple people on steam workshop having it aswell. It did not help deleting ACE and unsubcribing from steam. The github version does the same. ace_arsenal.pbo or ace_particles.pbo is outdated.

**Comment by dedmen** [comment]
Atleast one more report about this from Arma Discord.

**Comment by dedmen** [comment]
https://www.reddit.com/r/arma/comments/7m28y2/arma_3_editor_ace_error/

**Comment by jonpas** [comment]
I don't see a single RPT anywhere, we can't do anything without them.

**Comment by commy2** [comment]
No false positives. Show the RPT and the error will be visible for everyone knowing what to look for. Also, the Steam Workshop is bad and unreliable, and that's is nothing new or something we can fix. Ask Valve. 
Close?

**Comment by SilentSpike** [comment]
I'd say if there's no further info provided in about a week we can close it.

**Comment by Triiick** [comment]
Whats a RPT? Ill gladly show it but no idea what it is :)

**Comment by Dystopian** [comment]
https://community.bistudio.com/wiki/Crash_Files#Arma_3

**Comment by Triiick** [comment]
Here is my RPT file. It says atleast 2 times that the ace_arsenal.pbo is outdated!
https://pastebin.com/UVjBBqk5

**Comment by Dystopian** [comment]
your ACE is in `D:\Program Files x86\Steam\steamapps\common\Arma 3\!Workshop\@FK - ACE3` and old arsenal is in `D:\Program Files x86\Steam\steamapps\common\Arma 3\!Workshop\@ace\addons\ace_arsenal.pbo`

**Comment by Triiick** [comment]
Yes what does this mean?

**Comment by Dystopian** [comment]
use only one of them not both simultaneously

**Comment by Triiick** [comment]
So if i delete the old ace_arsenal.pbo will the mod pack still work normally? Or should i delete the old @ace folder?

**Comment by SilentSpike** [comment]
You need to launch arma only using one version of ace (or unsubscribe from one version to remove it from your system). If you delete the file steam will just replace it and you'll still be running two versions of ace anyway.

**Comment by Triiick** [comment]
Works now thanks mate
